### DISCUSSION TITLE
CLI option for HTTPS on local machine

### DISCUSSION THREAD

**Discussion by oleandreas** [loophole/cli#217] [General] [answered]
Hi

Thanks for a brilliant service!
I'd like to start loophole from command line for a site that already uses https. I se the option for this in the Gui tool, but haven't found out the option for cli.

Am I missing something, or is this only possible with the Gui tool?
![loophole-https](https://user-images.githubusercontent.com/5708376/223456618-d8abb121-27dc-4cb3-8e4d-7a3234c3bb59.png)

**Comment by 0x7f** [comment, marked as answer]
Hi @oleandreas 

very good point the [cli docs](https://loophole.cloud/docs/commands/http) do not list the [`--https`](https://github.com/loophole/cli/blob/1951585a14416aa0876174338ced5e2b85d0a145/cmd/http.go#L71) flag yet.

I will make sure to update them asap. Sorry for the trouble.

In the meantime, you can get more information about the support flags via `./loophole http --help`:

```
Exposes http server running locally, or on locally available machine to the public via loophole tunnel.

To expose server running locally on port 3000 simply use 'loophole http 3000'.
To expose port running on some local host e.g. 192.168.1.20 use 'loophole http <port> 192.168.1.20'

Usage:
  loophole http <port> [host] [flags]

Flags:
  -p, --basic-auth-password string   Basic authentication password to protect site with
  -u, --basic-auth-username string   Basic authentication username to protect site with
      --disable-old-ciphers          Disable TLS ciphers older than TLS1.2
      --disable-proxy-error-page     disable proxy error page and return 502 when your server is not available
  -h, --help                         help for http
      --hostname string              custom hostname you want to run service on
      --https                        use if your server is already using HTTPS
  -i, --identity-file string         private key path (default "/root/.loophole/.ssh/id_rsa")
      --path string                  specify path you wish to expose
      --qr                           use if you want a QR version of your url to be shown

Global Flags:
  -v, --verbose   verbose output
  ```

**Reply by oleandreas** [reply]
Excellent! Thx :)
